worker/include/RTC/ActiveSpeakerObserver.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef MS_RTC_ACTIVE_SPEAKER_OBSERVER_HPP
+#define MS_RTC_ACTIVE_SPEAKER_OBSERVER_HPP
+
+#include "RTC/RtpObserver.hpp"
+#include "handles/Timer.hpp"
+#include <json.hpp>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+// Implementation of Dominant Speaker Identification for Multipoint
+// Videoconferencing by Ilana Volfin and Israel Cohen. This
+// implementation uses the RTP Audio Level extension from RFC-6464
+// for the input signal.
+// This has been ported from DominantSpeakerIdentification.java in Jitsi.
+// https://github.com/jitsi/jitsi-utils/blob/master/src/main/java/org/jitsi/utils/dsi/DominantSpeakerIdentification.java
+namespace RTC
+{
+	class Speaker;
```

Comment:
Please let's define this `Speaker` class within the `ActiveSpeakerObserver` class. We don't want that this header file exposes a global `RTC::Speaker` class.